cypress/e2e/laboratory-preflight-script.cy.ts

```diff
@@ -1,7 +1,7 @@
 import { dedent } from '../support/testkit';
 
 const selectors = {
-  buttonPreflightScript: '[aria-label*="Preflight Script"]',
+  buttonPreflight: '[aria-label*="Preflight Script"]',
```

Comment:
Should we change the label value also?

Reply:
After looking into this, its coming from GraphiQL and it derives it from the given title  I think. I think changing that would enter into that product realm of something users see/could see, so will leave this as is for now.